components/Chat/ChatMessages.tsx

```
@@ -0,0 +1,144 @@
+            code: (props) => <CodeBlock {...props} />,
+          }}
+        >
+          {message.content}
+        </ReactMarkdown>
+      </div>
+      <div className="mt-2 mb-4 opacity-0 group-hover:opacity-100 transition-opacity flex items-center gap-4">
+        <button
+          onClick={() => copyToClipboard(message.content, message.id)}
+          className="text-xs px-2 py-2 rounded bg-gray-200 dark:bg-gray-800 text-gray-700 dark:text-gray-300 hover:bg-gray-300 dark:hover:bg-gray-700"
+          aria-label="Copy response"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)